terraform/main.tf

```diff
@@ -19,5 +19,5 @@ module "hello-world" {
   apigee_environment = var.apigee_environment
   proxy_type         = "sandbox"
   namespace          = var.namespace
-  make_api_product   = false
+  make_api_product   = true
```

Comment:
@Sourceless We set this value to false initially on our branch but after merging to master, release failed because now terraform is trying to delete the apiProduct and it can't:
```
resourceProductDelete error in product delete: DELETE https://api.enterprise.apigee.com/v1/o/nhsd-nonprod/apiproducts/hello-world-internal-dev: 400 Unable to delete ApiProduct as there are one or more apps associated with it
```
Shall we delete the `apiProduct` manually or do you think this we should go ahead with this change?

Reply:
keep it true, you'll need an api product for this service :)